src/services/utils.ts

```diff
@@ -10,4 +12,15 @@ function formatDateTime(date: Date) {
   return formatter.format(date);
 }
 
-export { formatDateTime };
+async function parseJsonFromBody(request: Request) {
```

Comment:
What does the function return? It needs to be `Record<string, any>`.

EDIT: actually double check the `.json()` method on the request. Hover on top of it, maybe it has a type, and we can use that.